### PR TITLE
doc: add documentation for allow_trigger_s5 in pm_notify_channel

### PR DESCRIPTION
**Pull request by peterfang**
Tracked-On: #6034
Signed-off-by: Peter Fang <peter.fang@intel.com>

### CODE REVIEW

File: doc/user-guides/acrn-dm-parameters.rst [hunk 1]
```diff
@@ -419,11 +419,13 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
 
    ACRN supports three channels: ``ioc``, ``power button`` and ``uart``.
 
+   For ``uart``, an additional option, ``,allow_trigger_s5``, can be added.
```
Comment (gvancuts) [inline doc/user-guides/acrn-dm-parameters.rst:422]: Can you also add a short explanation of what that parameter means?

**Review by gvancuts — COMMENTED** [review]
A bit minimalistic. It covers the basics as it adds the new parameter but we still need to further develop our documentation on how all of these components work together.

**Comment by wenlingz** [comment]
@peterfang doc PR don't need to create in release_2.5, please close it, thanks

**Comment by dbkinder** [comment]
> @peterfang doc PR don't need to create in release_2.5, please close it, thanks

Yes, that's correct.  We make doc updates on the master branch, and then we'll move all doc updates to the release_2.5 branch before we tag the final release.

**Comment by wenlingz** [comment]
@peterfang I close this PR

**Pull request closed without merging**